### DISCUSSION TITLE
response 429 error

### DISCUSSION THREAD

**Discussion by classcard0** [deepgram/community#846] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

![화면 캡처 2024-07-10 153637](https://github.com/deepgram/community/assets/165617802/c2d8a88c-4eed-4e33-8b6b-74d171cbd680)

We are using the Pay As You Go product and only use pre-recorded products. I know that the concurrent maximum is 100, but only 41 requests occurred within 10 seconds, as shown in the attached file.
However, error 429 is occurring.
The audio file for each request is about 3 to 4 seconds long, and the response takes about 2 to 3 seconds. In this case, the maximum concurrent value cannot exceed 100, but error 429 continues to occur. There is a service disruption, so please respond as quickly as possible.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?model=nova-2&filler_words=false&punctuate=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

c31c06e8-c9a1-435c-a81d-eedb33c8e5d5

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
@classcard0 Thank you for reporting this. I have looked into it and spoken to our engineers. This happened due to a temporary adjustment that was made to our rate limiting configuration. You should no longer see this issue happening, and I apologize for the disruption it made to your service. Please let me know if you still see the issue.

**Reply by classcard0** [reply]
Thank you for answer.
I'd like to know if this is just a mistake or if it's systematically and automatically limited.
In other words, I want to know if the problem can be reproduced.
Depending on the answer, I need to decide whether I need to come up with an alternative.

**Reply by SandraRodgers** [reply, marked as answer]
Of course, I'll explain what happened -  There was a configuration change in one of our datacenters that incorrectly calculated active requests. This caused some of your requests to be incorrectly rate limited.  The desired rate limiting behavior is to track 100 concurrent active requests, but in this case it had counted 100 requests/min. 

This misconfiguration has been resolved and you shouldn't see this behavior again.
